packages/teleport/src/Apps/AddApp/Manually/Manually.tsx

```diff
@@ -57,17 +57,17 @@ export default function Manually({ version, onClose }: Props) {
             <Text bold as="span">
               Step 3
             </Text>
-            {' - Generate a join token'}
-            <TextSelectCopy mt="2" text="tctl tokens add --type=app" />
+            {' - Generate an application join token'}
+            <TextSelectCopy mt="2" text="tctl tokens add --type=app --ttl=1h" />
           </Box>
           <Box>
             <Text bold as="span">
               Step 4
             </Text>
-            {` - Install Teleport on target server, and start it with the following parameters`}
+            {` - Install Teleport on target server, start Teleport with the following parameters`}
```

Comment:
Do we need to say anything about DNS and certificate?

Reply:
Also
`Install Teleport on target server, start Teleport with the following parameters`
to
`Install Teleport on target server, then start Teleport with the following parameters`

Reply:
> Do we need to say anything about DNS and certificate?

Cloud users won't need to make any changes to DNS since Applications will be on the subdomain of their cloud account. e.g. `internal-app.acme.teleport.sh` . Cloud will need obtain a cert for `*.acme.teleport.sh, acme.teleport.sh`

Re-naming: We've had a few versions. @klizhentas What do you think about

`Install the Teleport node service, then start Teleport with the following parameters`

Reply:
I think that's too many uses of "Teleport" in a row.

Also "node service" is a bit misleading when it's actually going to be running the Teleport app service; it should probably be called the `Teleport daemon`, `Teleport binary` or even `Teleport agent` which is probably the best description of what it actually does in this instance.

We have vastly overloaded the use of `service` as a noun :)

Reply:
> Cloud users won't need to make any changes to DNS since Applications will be on the subdomain of their cloud account. e.g. internal-app.acme.teleport.sh . 

@benarent there is no cloud version of teleport. We have E and OSS. So, what would be the manual steps for E users?

Reply:
@benarent could we combine E and cloud instructions?

```
1. If you are running Teleport onprem, make sure that Teleport proxy has the certificate [docs] installed. You can skip this step if you are running a cloud version.
```